app/models/repositories/EmailRepositoryResult.scala

```diff
@@ -17,6 +17,7 @@
 package models.repositories
 
 sealed trait EmailRepositoryResult
+
 case object SuccessfulEmail extends EmailRepositoryResult
+
 case object FailedToRetrieveEmail extends EmailRepositoryResult
```

Comment:
Can we add a new to the end of this file? it looks like it doesn't have to me. ignore this if its correct

Reply:
It already has a line break at the end